test/acceptance/replication/crud_test.go

```diff
@@ -333,9 +333,6 @@ func eventualReplicaCRUD(t *testing.T) {
 		helper.UpdateClass(t, pc)
 	})
 
-	// Ensure consistency on follower nodes by sleeping BEFORE shutting down the leader node
-	time.Sleep(3 * time.Second)
```

Comment:
This was tracked by https://semi-technology.atlassian.net/browse/WEAVIATE-701, it looks like the acceptance test for replication is still failing. Maybe we want to raise it up in standup and double check what is going on ?

Reply:
i got it fixed in previous run https://github.com/weaviate/weaviate/pull/4600/commits/9b8850867f4eeef85e4e141040233d2517c18862 , however i am truing now to remove more time.Sleep()